docs/data-guidelines.md

```diff
@@ -426,3 +426,21 @@ Note that APIs available on only _some_ types of workers are not considered glob
 - The `requestAnimationFrame()` function, available in `Window` and `DedicatedWorkerGlobalScope`.
 
 This guideline is based on a discussion in [#11518](https://github.com/mdn/browser-compat-data/pull/11518).
+
+## Setting `deprecated`
+
+Set `deprecated` to `true` to show that a feature has been formally discouraged from use.
```

Comment:
Hmm, I think it might be best to avoid the word outright since I'm using a bit… informally. Almost everything here is recognizing process but not necessarily depending on a formalism in a spec itself. I'm thinking this is the right way to go:

```suggestion
Set `deprecated` to `true` to show that a feature has been authoritatively discouraged from use.
```

Reply:
"Authoritatively" makes sense to me, yeah!